Swap Meta plumbing with morgue

cacogen opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

wesoda25 wanted someone to swap Meta's chem factory with the morgue. I thought it was a good idea that solved the problems I have with the current layout (i.e. needlessly big morgue, tacked-on chem factory).

### This PR:
- Replaces the Morgue, alcove in medbay with the NanoMed and Medical Surplus Storeroom below that with the chem factory
- Resizes the space now taken up by chem factory to be slightly larger
- Moves the Morgue to where the chem factory was and makes it considerably smaller
- Moves the maint bar to fill the rest of the gap left by the chem factory
- Moves the NanoMed alcove to the opposite side of hall
- Moves Medical Surplus Storeroom off maint above Virology, across from NanoMed alcove
- Adds two new maintenance Storage Rooms beneath robotics (needed to fill the space created by expanding the chem factory area downwards)

### Images
mapdiffbot shit the bed so after 9001 hours in ms paint

#### Before
![](https://i.imgur.com/h3sEdiu.jpg)

#### After
![](https://i.imgur.com/uksk1s0.jpg)
Xenobio bridge is back to what it was

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

- The chem factory is handy to medbay, and directly beneath chemistry
- The chem factory feels like an intrinsic part of medbay rather than a tacked-on afterthought
- The morgue is no longer needlessly big

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: cacogen
tweak: Swapped positions of Meta Morgue and chem factory
tweak: Resized the space now taken up by Meta chem factory to be slightly larger
tweak: Made Meta Morgue smaller
tweak: Moved Meta maint bar above the new Morgue where rest of the chem factory was
tweak: Moved Meta medbay alcove with NanoMed to opposite side of hall
tweak: Moved Meta Medical Surplus Storeroom above Virology, off maint across from alcove
add: Two new Meta maint Storage Rooms beneath robotics
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ArcaneMusic commented:
I'd say the new morgue could probably use a bit more trays than that personally, just take that extra little mait sliver and throw it into the morgue as well.
Also, there's good enough reason to wall chemistry with Rwalls, due to the nature of the more dangerous chemicals, and the chance of tiding players being able to break in easily.
(Also there's a hole from the main hall to the chem lab still)
But that's not important, actually.

What IS important is if we can get the old mait bar back with all the extra space that affords.

Buggy123 commented:
I like the idea (cloning removal aside), but I see a big issue: The new plumbing room is smaller. That's less than ideal considering how much space plumbing takes, I've seen lots of involved chem setups that take up the entirety of the current chemistry room.

Also, it feels a little silly to have a mini-chemistry room a few paces away from the real chemistry ('apothecary') room. It only made sense when plumbing was a entire department away from it, but if they're being moved so close together then chemistry and the mini-chemistry should be merged together.

Tlaltecuhtli commented:
yeah just expand plumbing area into the crappy locker room + chem room (they can use the maint door to go to chem
and you could also expand into the shitty south maint room that no one uses other than cultists 

also missing wall on the right?

necromanceranne commented:
Make sure roboticists, the chef and the chaplain can reach the new morgue.

wesoda25 commented:
Waiting for someone else to do what I’m too lazy to always works. Thanks cacogen 🙏

Dorsisdwarf commented:
While I hate to be awkward can I recommend changing medbay maint somewhat to avoid leaving a gigantic chemfactory-shaped hole?

cacogen commented:
I'll find the commit before the change and copy over the difference.

Sent via smoke signal

On Wed, 29 Jan 2020 at 1:29 AM, Dorsisdwarf <notifications@github.com>
wrote:

> While I hate to be awkward can I recommend changing medbay maint somewhat
> to avoid leaving a gigantic chemfactory-shaped hole?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/49054?email_source=notifications&email_token=AF7NO6T2U5ACPZ3HFUVEFTDRAAQKLA5CNFSM4KMMAIEKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEKDDY7A#issuecomment-579222652>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7NO6SRVHIXLL2ZZINZHDDRAAQKLANCNFSM4KMMAIEA>
> .
>

AWalton commented:
(This thread needs pictures).

wesoda25 commented:
You might wanna add a desk for the plumbing room, so that chemists know if anyone wants anything while setting stuff up.

Tlaltecuhtli commented:
can you add  a rwindow to the outside hall?

ShizCalev reviewed (dismissed):
move the xeno and blob spawners into the new morgue please.

kriskog commented:
This cant be merged before #48668 since it removes the cloners.

cacogen commented:
>There are no checks for this commit

kriskog reviewed (changes requested):
No chem dispenser in the new room, did you forget to add it?

cacogen commented:
@kriskog 
>Also, it feels a little silly to have a mini-chemistry room a few paces away from the real chemistry ('apothecary') room. It only made sense when plumbing was a entire department away from it, but if they're being moved so close together then chemistry and the mini-chemistry should be merged together.

I was following this part of @Buggy123's comment above and got rid of the Chemistry Lab they used to have given the proximity of the Pharmacy. Do you think they need one?

kriskog commented:
The pharmacy/chemistry shuffle feels really weird lately, where chemists are practically thrown out by busy doctors trying to make meds. I think giving them a degree of separation is good, assuming only the chemists have access to the plumbing room.

cacogen commented:
I added the chem dispenser back in. Looks a little unnatural. Also expanded the chem factory area downwards out of fear it would be too small. This meant gaps to fill. I could move medical surplus storage closer to medbay, and return the NanoMed alcove. But I need to work on the connection between virology and Medical Surplus Storage, as well as the new space beneath robotics. There's other things I need to tweak too. Plus I need to test it in-game and use the debug Mapping verbs.

Tlaltecuhtli commented:
>The pharmacy/chemistry shuffle feels really weird lately, where chemists are practically thrown out by busy doctors trying to make meds. I think giving them a degree of separation is good, assuming only the chemists have access to the plumbing room.

if you are full plumber you only need like 1-2 things from chemistry, if you dont want to do plumbing why did you not choose  to be a doc 🤔

cacogen commented:
i've been working on this for six hours... anyway i gave the chem factory its own chem dispenser and put it in front of the interior windoor they have. i think i'm gonna change the xenobio bridge back because it's ugly and i don't want that to undermine the rest of the PR. i tried a 3 wide bridge but it took up a lot of space for what it was and i figured people wouldn't like that. what else? oh, all the map errors for the parts i've changed are fixed. all that remains is what people think about the design and the functionality (e.g. anything else missing? anything unnecessary?)

kriskog commented:
cmon mapdiff

kriskog commented:
Ugh fuck you too then mapdiffbot.

kriskog reviewed (dismissed):
Atomize your PR please. The medbay thing is fine, but Im not a fan of the unrelated xenobio change.

cacogen committed:
Swap Meta's plumbing with morgue

cacogen committed:
puts the bridge back to what it was

cacogen commented:
I remember now I changed it because there was no longer a gap between the external wall of robotics maint and the glass of the bridge and two solid turfs beside each other can look bad so I opted to make the bridge wider because of multiple experiences with the awkwardness of passing mechs and simple mobs on it. Anyway, I changed it back. Not gonna update the picture in OP because it took too long to make.

kriskog commented:
Right, I didnt notice those changes since mapdiff isnt working and I thought it said maint robo in the changelog. Why are you changing stuff on that side anyway? Really should have been done in a separate PR.

Also making new commits isnt a crime, and we squash anyway. Mapdiffbot doesnt seem to like it when you force push the "same" commit.

cacogen commented:
Expanding the chem factory downwards meant extra space I had to fill. On that side as well.

kriskog commented:
Oh shit you made the whole station longer, yeah I see it now. This is why mapdiff is super useful. I need a while to consider this one then.

kriskog reviewed (changes requested):
![image](https://user-images.githubusercontent.com/33292112/74241134-ad769500-4cdb-11ea-9dd7-aa11f5fb10a2.png)
Wrong area in the middle here

![image](https://user-images.githubusercontent.com/33292112/74243787-a43bf700-4ce0-11ea-8177-4f1292e0badd.png)
Cables here

![image](https://user-images.githubusercontent.com/33292112/74243930-e7966580-4ce0-11ea-9984-7429bf07a573.png)
This junction is reversed

cacogen commented:
Forgot to test disposals... The cables are to electrify those windows. I'll fix that area.

cacogen committed:
fixes area, disposal junction

now to test

cacogen committed:
- add three firedoors and a fire alarm in main hall outside chem factory
- remove decals from abandoned airlocks to new maint rooms
- simplify disposal routes from bins in alcove and chem factory to trunk
- moves scrubber and vent in chem factory to be beside each other and further from others, using less pipes, looking neater and hopefully making the air alarm more effective

cacogen committed:
- use flip junction outside chem factory disposal
- fix camera tag duplication in clinic
- move medbay storage camera off airlock
- test cameras (no blindspots, no camera tag dupes)
- fix disconnected scrubber and air supply pipes in clinic
- test pipes (nothing disconnected, no extra pipes)
- test power (no missing wires, no extra wires)
- check intercom range display (have enough)
- check no roundstart ATs
- test areas (got everything they need)

cacogen commented:
i also tested disposals and it goes the right way now

kriskog reviewed (approved):
Ah, didnt see the windows there.

Looks okay now
